sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -484,6 +478,12 @@ export interface ServiceBusSessionReceiver<ReceivedMessageT extends ServiceBusRe
     };
 }
 
+// @public
+export interface ServiceBusSessionReceiverOptions<ReceiveModeT extends ReceiveMode> extends OperationOptionsBase {
+    maxAutoRenewLockDurationInMs?: number;
```

Comment:
We just noticed that we are not consistent with the naming here. Can you rename this to `maxAutoLockRenewalDurationInMs` to match the similar property in `ServiceBusReceiverOptions`?